docs/openapi/components/schemas/credentials/DeMinimisExemptionCertificate.yml

```diff
@@ -0,0 +1,136 @@
+$linkedData:
+  term: DeMinimisExemptionCertificate
+  '@id': https://w3id.org/traceability#DeMinimisExemptionCertificate
+title: De Minimis Exemption Certificate
+description: Ecommerce import declaration under Tarif Act Section 321 amendment, providing for an administrative de minimis exemption from duty and taxes for shipments of merchandise (https://www.govinfo.gov/content/pkg/FR-2019-07-23/pdf/2019-15625.pdf).
+type: object
+properties:
+  '@context':
+    type: array
+    items:
+      type: string
+      const:
+        - https://www.w3.org/2018/credentials/v1
+        - https://w3id.org/traceability/v1
+  type:
+    type: array
+    items:
+      type: string
+      const:
+        - VerifiableCredential
+        - DeMinimisExemptionCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    $ref: ../common/DeMinimisExemption.yml
+  proof:
+    type: object
+additionalProperties: false
+required: []
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "did:key:z6MkrPPZ84figsK3ksWXntFKJGANiJWJYwcusLc9tEKEmgsF",
+    "type": [
+      "VerifiableCredential",
+      "DeMinimisExemptionCertificate"
+    ],
+    "name": "Ecommerce Import Certificate",
+    "issuanceDate": "2022-07-12T10:00:00Z",
+    "issuer": {
+      "type": "Entity",
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "name": "Everything Ecom",
+      "address": {
+        "type": "PostalAddress",
+        "streetAddress": "175 Khanh Hoi, Ward 3",
+        "addressLocality": "Tan Phu Dist.",
+        "addressCountry": "VN"
+      },
+      "entityType": "Organization"
+    },
+    "credentialSubject": {
+      "type": "DeMinimisExemption",
+      "originatorCode": "CBP0000288215",
+      "participantFilerType": "Online Marketplace",
+      "shipmentTrackingNumber": "SN1000228491",
+      "modeOfTransportation": "Air",
+      "shipmentInitiator": {
+        "type": ["Entity"],
+        "name": "Everything Ecom",
+        "address": {
+          "type": "PostalAddress",
+          "streetAddress": "175 Khanh Hoi, Ward 3",
+          "addressLocality": "Tan Phu Dist.",
+          "addressCountry": "VN"
+        },
+        "entityType": "Organization"
+      },
+      "seller": {
+        "type": "Entity",
+        "name": "Phongh",
+        "address": {
+          "type": "PostalAddress",
+          "streetAddress": "19 Giai Phong St., 395 Lane, Phuong Liet Ward",
+          "addressLocality": "Thanh Xuan Dist.",
+          "addressCountry": "VN"
+        },
+        "entityType": "Organization"
+      },
+      "buyer": {
+        "type": "Entity",
```

Comment:
No more using Entity, use Organization.

Reply:
I've changed some of these attributes (`shipmentInitiator` and `seller`), but `buyer` and `finalDeliverTo` will often be a person for De Minimis shipments. Remember, this schema is intended for duties exemption when the import is worth less than $800, specifically for orders made on foreign ecommerce platforms. I propose that we keep these as `Entity` - pls let me know if you still have objections.